run.py

```diff
@@ -320,14 +373,18 @@ def forecast_each_county(
         f"total cases: {total_cases} beds: {beds} pop: {population}"
     )
 
-    for i, intervention in enumerate(get_interventions()):
+    all_website_data = []
+    interventions = get_interventions()
+    for i, intervention in enumerate(interventions):
         results = model_state(cases, beds, population, intervention)
         website_data = prepare_data_for_website(
             results, cases, population, min_date, max_date, interval=4
         )
-
+        all_website_data.append(website_data)
         write_results(website_data, output_dir, f"{state}.{fips}.{i}.json")
 
+    return all_website_data
```

Comment:
You changed these functions to return this `all_website_data` thing, but I don't see where it is ultimately used.  I'm probably just confused.

Reply:
ah yeah, I'll also remove this until we actually need the output result.  It felt appropriate to have the function return the results of running the model, but don't yet need it for anything.